hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieCreateHandle.java

```diff
@@ -235,19 +235,18 @@ protected void setupWriteStatus() throws IOException {
     stat.setPrevCommit(HoodieWriteStat.NULL_COMMIT);
     stat.setFileId(writeStatus.getFileId());
     stat.setPath(new Path(config.getBasePath()), path);
-    stat.setTotalWriteBytes(computeTotalWriteBytes());
-    stat.setFileSizeInBytes(computeFileSizeInBytes());
     stat.setTotalWriteErrors(writeStatus.getTotalErrorRecords());
+
+    long totalWrittenBytes = computeTotalWrittenBytes();
+    stat.setTotalWriteBytes(totalWrittenBytes);
+    stat.setFileSizeInBytes(totalWrittenBytes);
+
     RuntimeStats runtimeStats = new RuntimeStats();
     runtimeStats.setTotalCreateTime(timer.endTimer());
     stat.setRuntimeStats(runtimeStats);
   }
 
-  protected long computeTotalWriteBytes() throws IOException {
-    return FSUtils.getFileSize(fs, path);
-  }
-
-  protected long computeFileSizeInBytes() throws IOException {
+  protected long computeTotalWrittenBytes() throws IOException {
```

Comment:
computeFileSizeInBytes is a better name here and matches the FSUtils.getFileSize.

It is an internal details that for CreateHandle the total written bytes will be equal to the file size.

Reply:
Good point